bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/ScriptEngineFactory.java

```diff
@@ -1,8 +1,8 @@
 /**
- * Copyright (c) 2014,2019 Contributors to the Eclipse Foundation
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
```

Comment:
Could you rebase your branch.
This change should already be part of master.

Reply:
I had put this in to remove the conflict, since I knew the header PR would be merged before mine, but I've rebased.